docs/07_tutorial4.md

```diff
@@ -155,3 +155,97 @@ So how can a machine learning model detect ink? In the electron microscope image
+cd Vesuvius-Grandprize-Winner
+```
+3. Download and install anaconda or miniconda from: https://www.anaconda.com/download
+
+4. Create a new Conda env and activate it
+
+```bash
+conda create -n vesuviusgp
+conda activate vesuviusgp
+```
+5. Install the requirements
+
+```bash
+pip install -r requirements.txt
+```
+
+At this point you have the model and all the requirements on your pc. The current inference and training scripts use relative paths, all starting from the directory from which you cloned the repository.
+
+What this means is that when you run the inference or training scripts, they are starting from that directory, and looking for the folders from there. 
+
+The following directory structure is required, unless you want to modify the paths in the script. Note that the segment name and the _mask.png and _inklabels.png must match.  
+.
+└── Vesuvius-Grandprize-Winner-main/
+    ├── eval_scrolls/
+    │   └── segment1/
+    │       ├── layers/
+    │       │   └── 00.tif
+    │       └── segment1_mask.png
+    └── train_scrolls/
+        └── segment1/
+            ├── layers/
+            │   └── 00.tif
+            ├── segment1_mask.png
+            └── segment1_inklabels.png
+
+For inference; 
+There are some defaults in the InferenceArgumentParser class, but it also accepts them as inputs
+
+```bash
+    segment_id: list[str] =['20230925002745'] #this accepts multiple segments, just keep in mind all these must exist in eval_scrolls, unless you change the segment_path
+    segment_path:str='./eval_scrolls' #the path to the segments
+    model_path:str= 'outputs/vesuvius/pretraining_all/vesuvius-models/valid_20230827161847_0_fr_i3depoch=7.ckpt' #the path to the checkpoint you downloaded or created and want to use for inference
+    out_path:str="" #path to output predictions
+    stride: int = 2 #the amount of pixels to skip when sliding the window, default is 2, you can run this up to 32 with no real difference in quality but a substantial increase to speed
+    start_idx:int=15 
+    workers: int = 4 #number of threads to use, increase or decrease depending on your specs, not to exceed number of threads avail
+    batch_size: int = 512 #increase or decrease depending on hardware, would have some difficulty above 512 on regular consumer hardware. if you get CUDA: out of memoery errors, lower this or increase stride, or both
+    size:int=64 #window size
+    reverse:int=0 #use to reverse the order of the layers during inference
+    device:str='cuda' #change to cpu or mps if on device with no gpu or apple device, respectively
+```
+these all can be passed as arguments. ex: `python inference_timesformer.py --segment_id 20231210121321 20231221180251 --segment_path $(pwd)/train_scrolls --model_path timesformer_wild15_20230702185753_0_fr_i3depoch=12.ckpt`
+
+advanced: to change the number of layers the model runs inference on, you must change the following:
+
+```bash
+
+    in the read_image_mask function, where it is defined
```

Comment:
(I will assume these seperated lines are intended to and will be shown as comments even without a hashtag or similar.)